app/views/group_bills/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Group bills</h1>
+
+<div id="group_bills">
+  <% @group_bills.each do |group_bill| %>
+    <%= render group_bill %>
+    <p>
+      <%= link_to "Show this group bill", group_bill %>
+    </p>
+  <% end %>
+</div>
+
+<%= link_to "New group bill", new_group_bill_path %>
```

Comment:
- [ ] You did a great job working on your app however you are required to match the given design as much as possible. Kindly  work on the highlighted section to match the design.


Your design:


<img width="442" alt="Screenshot 2023-09-15 at 5 48 42 PM" src="https://github.com/musangamfure/BudgetApp/assets/53347966/3c89b440-6d6d-46db-961e-c19585e98cf6">



The provided Design:

<img width="309" alt="transactions_list (1)" src="https://github.com/musangamfure/BudgetApp/assets/53347966/c82659ac-36df-45fd-925d-c91a2396e855">

Reply:
Hello @algerina,

Thank you so much for your comment. But I would like to get clarification on what you want me to change. As you are pointing out in your screenshots, there are two pages. one is for individual groups, and the other is for all the transactions/groups. And I think they are all visible to  on my app.

here is my app:
<img width="256" alt="groups" src="https://github.com/musangamfure/BudgetApp/assets/123179856/8a546436-0666-4894-8e24-a4431a3b6bca">

Reply:
- Hello @musangamfure, you have done well so far, but according to the previous reviewer, please style the transactions page as it is in the design template. From what the previous reviewer marked in the screenshot, the marked section, i.e the name of the category and the total amount, should be directly under the blue navbar. Please fix this